symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -165,59 +170,67 @@ public TemplateEngine templates() {
    * @return a {@link V4Message} object containing the details of the sent message
    * @see <a href="https://developers.symphony.com/restapi/reference#create-message-v4">Create Message v4</a>
    */
+  @Deprecated
   public V4Message send(@Nonnull V4Stream stream, @Nonnull String message) {
     return send(stream.getStreamId(), message);
   }
 
   /**
-   * Sends a templated to the stream ID of the passed {@link V4Stream} object.
+   * Sends a message to the stream ID passed in parameter.
    *
-   * @param stream     the stream to send the message to
-   * @param template   the template name to be used to produce the message
-   * @param parameters the parameters to pass to the template to produce the message to be sent
+   * @param streamId the ID of the stream to send the message to
+   * @param message  the message payload in MessageML
```

Comment:
Use Javadoc [`@deprecated`](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag) tag to mention that this method will be replaced by `send(String, Message)`